packages/p2p/src/stores/my-profile-store.js

```diff
@@ -813,4 +842,24 @@ export default class MyProfileStore extends BaseStore {
     setTradePartnersList(trade_partners_list) {
         this.trade_partners_list = trade_partners_list;
     }
+
+    upgradeDailyLimit() {
+        const { general_store } = this.root_store;
+
+        requestWS({ p2p_advertiser_update: 1, upgrade_limits: 1 }).then(response => {
+            if (response) {
+                this.setIsLoadingModalOpen(false);
+
+                if (response.error) {
+                    clearTimeout(wait);
+                    const wait = setTimeout(() => this.setIsErrorModalOpen(true), 250);
+                } else {
+                    clearTimeout(wait);
+                    const wait = setTimeout(() => this.setIsDailyLimitSuccessModalOpen(true), 250);
```

Comment:
is the setTimeout used to prevent any overlapping of modals here?

Reply:
@ameerul-deriv yes